ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/bgp-peer-groups-1.cfg

```diff
@@ -136,7 +136,5 @@ router bgp 65001
       no neighbor EVPN-OVERLAY-PEERS activate
       neighbor IPv4-UNDERLAY-PEERS activate
       neighbor MLAG-IPv4-UNDERLAY-PEER activate
-   !
-   address-family vpn-ipv4
 !
```

Comment:
This change is a bugfix. We should not render useless config. The change is caused by the new `_strip_empties` call where we remove all empty lists/dicts/None value from the output.